### PR TITLE
MS DNS Winlogbeat Mapping

### PR DESCRIPTION
**Pull request by eepstain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-7642)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftDNS/README.md [hunk 1]
```diff
@@ -20,14 +18,25 @@ For the Filebeat collector, use the following option to collect events from the
...
 Use the information described [here](https://docs.paloaltonetworks.com/cortex/cortex-xdr/cortex-xdr-pro-admin/cortex-xdr-collectors/xdr-collector-datasets#id7f0fcd4d-b019-4959-a43a-40b03db8a8b2).
 
 You can configure the vendor and product by replacing [vendor]\_[product]\_raw with *microsoft_dns_raw*.
 
 When configuring the instance, you should use a yml file that configures the vendor and product, as shown in the below configuration for the Microsoft DNS product.
- 
-For **XSIAM version 1.2** only, copy and paste the below in the *Filebeat Configuration File* section (inside the relevant profile under the *XDR Collectors Profiles*).
+
+**Pay Attention**:
+When using this pack their are two integrations available for it.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDNS/README.md:28]:
```suggestion
When using this pack there are two integrations available for it.
```

File: Packs/MicrosoftDNS/README.md [hunk 2]
```diff
@@ -20,14 +18,25 @@ For the Filebeat collector, use the following option to collect events from the
...
+
+* ***As enrichment, forwarding DNS Audit logs is supported via Winlogbeat***
+
+The first, is via Filebeat for DNS Debug logs.
+The second, is via Winlogbeat for DNS Audit logs.
+
+Follow these steps in order to configure the XDR Collector:
+1. The implementation of the Collector for both of the methods requires to create a [Profile](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Add-an-XDR-Collector-Profile-for-Windows) for each integration.
+   * For **XSIAM version 1.2** only, copy and paste the below in the *Filebeat Configuration File* section (inside the relevant profile under the *XDR Collectors Profiles*).
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDNS/README.md:37]:
```suggestion
   * For **XSIAM version 1.2** only, in the relevant profile under the *XDR Collectors Profiles*, copy and paste the information from the [Filebeat Configuration File](#filebeat-configuration-file) section.
```

File: Packs/MicrosoftDNS/README.md [hunk 3]
```diff
@@ -44,4 +53,17 @@ filebeat.inputs:
...
+  - name: Microsoft-Windows-DNSServer/Audit
+    processors: 
+      - add_fields: 
+          fields: 
+            vendor: microsoft
+            product: dns
+    id: dns-audit-logs
+```
+
+For **XSIAM version 1.3** and above, please use the built-in YAML template provided within the XDR collector configuration.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDNS/README.md:69]:
```suggestion
For **XSIAM version 1.3** and above, use the built-in YAML template provided within the XDR collector configuration.
```

File: Packs/MicrosoftDNS/ReleaseNotes/1_0_6.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+
+#### Modeling Rules
+
+##### Microsoft DNS
+
+Updated the Modeling Rule logic to include the mapping of additional fields from DNS events via Winlogbeat (In addition to Filebeat).
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDNS/ReleaseNotes/1_0_6.md:6]:
```suggestion
Updated the Modeling Rule logic to include the mapping of additional fields from DNS events via Winlogbeat (in addition to Filebeat).
```

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.

**Review by evisochek — APPROVED** [review]

**Pull request merged**